client/components/plans/plans-compare/index.jsx

```diff
@@ -74,6 +73,13 @@ const wordAdsFeature = {
 const PlansCompare = React.createClass( {
 	mixins: [ observe( 'features' ) ],
 
+	componentWillMount() {
+		this.setFreePlan = () => this.setPlan( 'free' );
+		this.setPersonalPlan = () => this.setPlan( 'personal' );
+		this.setPremiumPlan = () => this.setPlan( 'premium' );
+		this.setBusinessPlan = () => this.setPlan( 'business' );
```

Comment:
I need this so that I don't create new fns each time a component is rendered (ie don't use `bind` or arrow fns). I wonder, isn't there a nicer way to accomplish the same?